hw11.py

```diff
@@ -0,0 +1,53 @@
+import time
+
+
+def name_and_time(func):
+    print(f'You called function "{func.__name__}" on {time.ctime()}')
+
+    def inner_deco_func():
```

Comment:
Add here *args and **kwargs arguments and pass them to the `func` call to be able to decorate functions that take arguments